website/_kb/data-sources/supported-data-sources.md

```diff
@@ -31,15 +33,24 @@ Greenplum  |  ✅  |  ✅
 Hive  |  ✅  |  ✅  
 Impala  |  ✅  |  ✅  
 InfluxDB  |  ✅  |  ✅  
+JQL for Jira  |  ✅  |  ✅  
```

Comment:
This is already in the list.

Reply:
Fixed. The list wasn't properly alphabetized so I didn't see Jira on the list. Fixed in my latest commit.